src/components/Table/Table.js

```diff
@@ -0,0 +1,137 @@
+import _ from "lodash";
```

Comment:
please call this component SongsTableComponent. name Table is too generic. Do not forget to rename css classnames and files